index.js

```diff
@@ -335,8 +335,12 @@ morgan.token('user-agent', function getUserAgentToken (req) {
  */
 
 morgan.token('req', function getRequestToken (req, res, field) {
+  if (!field) {
+    throw new Error('req is missing an argument')
```

Comment:
So this just suggests changing the argument type and the wording to match (a) the documentation in morgan (calling the argument header) and (b) the wording that this type of error usually is in express projects.

```suggestion
    throw new TypeError('header argument is required to :req token')
```